examples/async_transcribe_local_media_file.js

```diff
@@ -9,7 +9,7 @@ const token = require('./config/config.json').access_token;
     // Get account details
     var account = await client.getAccount();
     console.log(`Account: ${account.email}`);
-    console.log(`Balance: ${account.balance_seconds} seconds`);
+    console.log(`Credits remaining: ${account.balance_seconds} seconds`);
```

Comment:
Should we change `balance_seconds` to `credits_seconds` or something similar?

Reply:
I believe not, we only want to change strings and descriptions, not any code.

Reply:
Yes. No breaking changes. thanks